src/plugins/schema-filter/index.ts

```diff
@@ -0,0 +1,88 @@
+import { LegacySettings } from '../../browser'
+import { Context } from '../../core/context'
+import { PlanEvent } from '../../core/events/interfaces'
+import { Plugin } from '../../core/plugin'
+import { RemotePlugin } from '../remote-loader'
+
+function disabledActionDestinations(
+  plan: PlanEvent,
+  settings: LegacySettings
+): { [destination: string]: string[] } {
+  if (!plan || !Object.keys(plan)) {
+    return {}
+  }
+
+  const disabledIntegrations = Object.keys(plan.integrations).filter(
+    (i) => plan.integrations[i] === false
+  )
+
+  // This accounts for cases like Fullstory, where the settings.integrations
+  // contains a "Fullstory" object but settings.remotePlugins contains "Fullstory (Actions)"
+  const disabledRemotePlugins: string[] = []
+  ;(settings.remotePlugins ?? []).forEach((p: RemotePlugin) => {
+    disabledIntegrations.forEach((int) => {
+      if (p.name.includes(int) || int.includes(p.name)) {
```

Comment:
There is a case already where the field on the `plan` object is different from `integrations`, but they're related to the same action destination.

> For example, if someone disabled Braze Actions from the schema filters, then we will get Braze Web Mode (Actions) in both the plan object, and the remote plugins list.

The above is true for Braze, but not for Fullstory. When Fullstory was registered for the first time, instead of registering it with the name `Fullstory (Actions)`, it was registered as `Fullstory`, creating that difference between the `integrations` and the `plan` object.